CHANGELOG.md

```diff
@@ -3,7 +3,82 @@
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.43.0
+
+This release contains a number of fixes and enhancements.
+
+### Tooling, SDK, and Runtime
+
+- Add line count to test coverage report (#4843) (authored by @FarisR99)
+- build: run 'make generate' in release container
+- Bundles fixes, changes:
+  - plugins: accept any HTTP 2xx for status and logs (#4897) (authored by @lvisterin)
+  - runtime: Treat other Unix-es like macos (#4931) (authored by @iamleot)
+  - bundle: Add support for .yml files in opa bundle command (#4882) ([#4859](https://github.com/open-policy-agent/opa/issues/4859)) authored by @Joffref reported by @rdrgmnzsakt
+  - plugins/bundle: use unique temporary files (#4786) ([#4782](https://github.com/open-policy-agent/opa/issues/4782)) authored by @FredrikAppelros reported by @FredrikAppelros
+  - server: check old policy path for bundle ownership (#4847) ([#4846](https://github.com/open-policy-agent/opa/issues/4846))
+  - storage+bundle: Clean old bundle data before new activation (#4944) ([#4940](https://github.com/open-policy-agent/opa/issues/4940))
+  - bundle: Normalize paths before bundle root check
+- Storage fixes:
+  - storage/disk: Unlock mutex on error
+  - storage/inmem: Create path if does not exist during truncate (#4853)
+  - storage/disk: make symlinks work with relative paths (#4870) ([#4869](https://github.com/open-policy-agent/opa/issues/4869))
+  - storage/inmem: Avoid unnecessary read operation ([#4898](https://github.com/open-policy-agent/opa/issues/4898))
+
+### Rego and Topdown
+
+- ast/compiler: allow retaining parsed modules (#4921) ([#4910](https://github.com/open-policy-agent/opa/issues/4910))
+- ast/term: Make Object key sorting lazy. (#4830) ([#4625](https://github.com/open-policy-agent/opa/issues/4625))
+- object.subset: support array + set combination (#4865) ([#4858](https://github.com/open-policy-agent/opa/issues/4858)) authored by @x-color
+- add compiler.Compiler and compiler.optimizer WithEnablePrintStatements option and pass through to ast.Compiler (authored by @kevinstyra)
+- Topdown fixes:
+  - ast/builtins: fix 'type_name' type and docs
+  - topdown/copypropagation: keep refs into livevars (#4936) ([#4848](https://github.com/open-policy-agent/opa/issues/4848)) reported by @johanneskra
+  - topdown/parse_units: Use big.Rat for units parsing. (#4857) ([#4856](https://github.com/open-policy-agent/opa/issues/4856)) reported by @tmos22
+  - topdown: is_valid functions no longer error. (#4844) ([#4760](https://github.com/open-policy-agent/opa/issues/4760))
+  - topdown: support glob.match without delimiters (#4933) ([#4923](https://github.com/open-policy-agent/opa/issues/4923)) authored by @vinhph0906 reported by @vinhph0906
+- ast: move schema tests from compile_test.go to schema_test.go
+
+### Documentation
+
+ - docs/bundles: add bundle CLI command documentation (#4881) ([#3831](https://github.com/open-policy-agent/opa/issues/3831)) authored by @Joffref
```

Comment:
I usually remove the `(#...)` bit -- it's a thing github adds to commits when using squash and merge.

Reply:
Done. :+1: